api/get_video_info.py

```
@@ -2,18 +2,21 @@
 import json
 import sys
 import os
+import traceback
```

Ruff (0.11.9)

`traceback` imported but unused

Remove unused import: `traceback`

(F401)